lib/vmdb/settings/database_source.rb

```diff
@@ -3,23 +3,57 @@ class Settings
     class DatabaseSource
       include Vmdb::Logging
 
-      attr_reader :resource
+      attr_reader :resource_instance, :settings_holder_class
```

Comment:
Why rename to resource_instance?  I see `resource` and that implies that it's an instance to me already, so seems redudant.